pkg/path/parser/node.go

```diff
@@ -0,0 +1,61 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package parser
+
+type NodeType string
```

Comment:
What is the relationship between these structures and those being added in https://github.com/open-policy-agent/gatekeeper/pull/926 ? Do you expect some kind of conversion?

Reply:
They represent the same thing - these names made more sense to my mind while I was coding - I'm ok with renaming on either side depending on what everyone likes better :)

Reply:
Personally the only objection I have is `Value` (commented on below)

Reply:
Renamed. We can keep workshopping this - it might make more sense as we write code that consumes the paths.

Reply:
+1